File: backend_healthconnect/src/main/java/com/backend_healthconnect/model/visitaDTO.java
package com.backend_healthconnect.model;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import java.time.LocalDate;
import java.time.LocalDateTime;

@AllArgsConstructor
@NoArgsConstructor
@Getter
@Setter
public class visitaDTO {
    Long id;
    prenotazioneDTO prenotazione;
    utenteDTO paziente;
    utenteDTO medico;
    String diagnosi;
    String noteMedico;
    LocalDateTime dataVisita;

}

File: frontend_healthconnect/src/app/model/visitaDTO.ts
import {prenotazioneDTO} from './prenotazioneDTO';
import {utenteDTO} from './utenteDTO';

export interface VisitaDTO {
  id: number;
  prenotazione: prenotazioneDTO;
  paziente: utenteDTO;
  medico: utenteDTO;
  diagnosi: string;
  note: string;
  data: string;
}
